charlesinto/iReporter
# feat(endpoint): Update Intervention status

#### What does this PR do?

- Update intervention status

#### Description of Task to be completed?
- Implement JWT authentication
- Verify User is Admin
- Update Status

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `PATCH` request to `http://localhost:7000/api/v1/interventions/:id/status`
- You should receive a successful response

#### Any background context you want to provide?
The request should be a structure like:

`{status: "string"}`

`

#### What are the relevant pivotal tracker stories?
[#162597939](https://www.pivotaltracker.com/story/show/162597939)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/26615085/49847243-689d5880-fdcf-11e8-91d8-3cf9590375fd.png)






#### Questions:
N/A
